lib/directory_diff/transform.rb

```diff
@@ -40,7 +40,10 @@ def process_employee(email, assistant_owner)
 
       if new_employee.nil?
         add_transform(:delete, old_employee)
-        assistant_owner[3] = nil
+        assistants_string = assistant_owner[3].to_s.split(",").reject do |assistant|
+                              assistant == email
+                            end.join(",")
+        assistant_owner[3] = assistants_string == "" ? nil : assistants_string
```

Comment:
Oh yeah forgot about that. Merge away unless you want to add more test around that where every email is missing. Does it work for an employee that was being updated instead of insert?